### PR TITLE
Fix gh-1472 & gh-1473: FAQ Updates

### LINKED ISSUE #1472: Typo on FAQ page "For more more questions about Teacher Accounts..."
Remove the extra "more" in "For more more questions about Teacher Accounts, see the Teacher Account FAQ" at the bottom of the FAQ page (https://scratch.mit.edu/info/faq).

### PR DESCRIPTION
**Pull request by St19Galla**
This fixes #1472 and #1473 by deleting the second case of the word "more" and by changing the link text do "download Scratch 2.0".

/cc @jwzimmer @thisandagain @mewtaylor @rschamp

### CODE REVIEW

File: src/views/faq/l10n.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
...
     "faq.offlineTitle":"Do you have a downloadable version so I can create and view projects offline?",
-    "faq.offlineBody":"The Scratch 2 offline editor allows you to create Scratch projects without an internet connection. You can download Scratch 2 from the <a href=\"/scratch2download/\">website</a>. You can also still use <a href=\"/scratch_1.4\">Scratch 1.4</a>. Note: You can have both Scratch 1.4 and 2 on your computer.",
+    "faq.offlineBody":"The Scratch 2 offline editor allows you to create Scratch projects without an internet connection. You can <a href=\"/scratch2download/\">download Scratch 2.0</a> from the website. You can also still use <a href=\"/scratch_1.4\">Scratch 1.4</a>. Note: You can have both Scratch 1.4 and 2 on your computer.",
```
Comment (thisandagain) [inline src/views/faq/l10n.json]: Can we update the multiple instances of "Scratch 2" to be "Scratch 2.0" in this string?

Comment (St19Galla) [inline src/views/faq/l10n.json]: @thisandagain For sure! Should I also update all the other instances on the faq page?

Comment (thisandagain) [inline src/views/faq/l10n.json]: @St19Galla Yes please!

**Comment by St19Galla** [comment]
@thisandagain done!

**Comment by mewtaylor** [comment]
On Chrome, OSX 10.12:
* The extra `more` is gone at the bottom of the FAQ
* Text for download says `Scratch 2.0`
* `download Scratch 2.0` is now the link text
* the link points to the download page

The text for the FAQ page earlier on says `Scratch 2.0`, but the header for the `/download` page says `Scratch 2` still. @thisandagain – in [your comment](https://github.com/LLK/scratch-www/issues/1473#issuecomment-323840238), were you referring to the scope for this also including changing the heading on the `/download` page, or just the FAQ page?

**Comment by mewtaylor** [comment]
The title is fixed in https://github.com/LLK/scratch-www/commit/38d79df894403e5ad88dc67fd5656585d3828bea

**Pull request merged**